pontoon/base/static/js/fluent_interface.js

```diff
@@ -228,8 +253,8 @@ var Pontoon = (function (my) {
       // Render SelectExpression
       if (element.expression && element.expression.type === 'SelectExpression') {
```

Comment:
Why does this check for `element.expression` first?

Reply:
We don't always have `element.expression.type`, e.g.:
https://github.com/mozilla-l10n/pontoon-ftl/blob/master/en-US/demo.ftl#L14

Reply:
Do you mean the fact that 1) this is an old 0.4 syntax? Or the fact that 2) these are `TextElements`?

1. I think we shouldn't have any 0.4 strings left anywhere across current Mozilla projects. In fact, `fluent-syntax` 0.8.0 removed the Syntax 0.4 support. This string will now parse as `Junk`. It might be worth changing the `demo.ftl` file.

2. `isSimpleElement` should catch `TextElements`. In fact, shouldn't there be an early return in `renderOriginalElements` just before `// Render SelectExpression`? Or at least I'd suggest adding a `if (element.type === "Placeable")` test there. All `Placeables` are guaranteed to have the `expression` field.

Reply:
Oh, right... Let me first fix the demo.ftl.